apps/opencs/view/render/terrainselection.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef CSV_RENDER_TERRAINSELECTION_H
+#define CSV_RENDER_TERRAINSELECTION_H
+
+#include <utility>
+#include <vector>
+
+#include <osg/Vec3d>
+#include <osg/ref_ptr>
+#include <osg/PositionAttitudeTransform>
+
+#include <components/esm/loadland.hpp>
+#include "../../model/world/cellcoordinates.hpp"
```

Comment:
I think this should probably be a full path instead of a `..`-based one.

Reply:
Nope, that is how we normally do includes.